source/localizable/models/creating-updating-and-deleting-records.md

```diff
@@ -151,14 +151,12 @@ deletion can then be persisted using `save()`.  Alternatively, you can use
 the `destroyRecord` method to delete and persist at the same time.
 
 ```js
-store.findRecord('post', 1).then(function(post) {
```

Comment:
Lets keep the `findRecord` because the record to be deleted may or may not be in the store but change the example to use the `backgroundReload` to disable the potential for a `scheduleFetch()`.

`store.findRecord('post', 1, { backgroundReload: false }).then(function(post) {`